src/doom/p_user.c

```diff
@@ -34,7 +34,12 @@
 
 
 // Index of the special effects (INVUL inverse) map.
+#ifndef CRISPY_TRUECOLOR
 #define INVERSECOLORMAP		32
+#else
+// [crispy] parameterized for smooth diminishing lighting
+int INVERSECOLORMAP;
```

Comment:
Let's just turn it into an `int` everywhere.

Reply:
Nope, wait. Let's turn `NUMCOLORMAPS` into an `int` and have `#define INVERSECOLORMAP NUMCOLORMAPS` everywhere.